tests/test_models.py

```diff
@@ -433,3 +433,9 @@ def test_display_text_block(test_document, test_spans_valid):
     assert isinstance(block_html, str)
     assert len(block_html) > 0
     assert block_html.startswith("<div")
+
+
+def test_text_block_hashable(test_document):
+    doc = test_document
+
+    set(doc.text_blocks)
```

Comment:
Assuming this test isn't complete? Do we not need to assert the hashes text block as an int of length 18?

Reply:
I was meaning to test that the hashing works by set creation, but I do see it's not sufficient. I've added more asserts with respect to equality of objects, and the type returned. I don't think the length of the int is fixed though.